### DISCUSSION TITLE
how to re-use markdown code across pages?

### DISCUSSION THREAD

**Discussion by borango** [observablehq/framework#1428] [Q&A] [answered]
In my Observable Cloud Notebooks I have various named markdown cells which I can import in other Notebooks.

In Observable Framework I have not found a way to re-use sections of markdown code. If I bind the markdown text, which would be rendered to HTML if written literally on a .md page, to a variable `note`, then displaying `note` will show the literal markdown code instead of the formatted HTML.

It feels like the Framework standard library is lacking md() - https://observablehq.com/documentation/misc/standard-library#md-string

What would be the equivalent in Framework?

**Comment by Fil** [comment, marked as answer]
See https://github.com/observablehq/framework/issues/895 and https://observablehq.observablehq.cloud/pangea/party/markdown-it

**Reply by borango** [reply]
Thanks @Fil
